### ISSUE TITLE
Add Basic Page

### ISSUE THREAD

**Issue by bgub** [nebrelbug/awesome-for-beginners#3]
I think now would be a good time to get started working on at least a working title page :)

**Comment by Meikay** [comment]
@nebrelbug Hi I'm new, is this assigned to anyone? Can I take this on?

**Comment by bgub** [comment]
@Meikay that would be great! Thanks!

**Comment by Meikay** [comment]
@nebrelbug just to be clear, did you mean a front page for the website?

**Comment by Meikay** [comment]
@nebrelbug Hi, I just wanted to update you. I am currently working on the front page, is there anything specific you would like me to add to it?

**Comment by nxv1317** [comment]
@Meikay Hi, I would also like to help out with this page. Would you mind working together on this?

**Comment by Meikay** [comment]
@nxv1317 Yeah that would be great! Just trying to figure out the details, do you happen to know?

**Comment by nxv1317** [comment]
Unfortunately no, looking back that hasnt been made clear yet

**Comment by nxv1317** [comment]
@nebrelbug when possible could you please update us with what info to add, thanks

**Comment by bgub** [comment]
@Meikay and @nxv1317, I think just a basic page with a list of projects would be great to start with. Sorry I took so long to respond!

**Comment by nxv1317** [comment]
@nebrelbug thanks for the reply. @meikay you said you are currently working on the front page could you share what you have produced so far with me, then we can take it from there thanks

**Comment by Meikay** [comment]
@nxv1317 How do you prefer I share the code with you?

**Comment by nxv1317** [comment]
@meikay maybe put it into a folder and share it on here or personally message me

**Comment by bgub** [comment]
@Meikay and @nxv1317 I would suggest working with @ascii-dev and all of the other people who commented wanting to help here: [https://github.com/MunGell/awesome-for-beginners/issues/199](https://github.com/MunGell/awesome-for-beginners/issues/199)

**Comment by Meikay** [comment]
@nxv1317 @nebrelbug Hi, this is what I have so far, I didn't do much since I wanted to get input from everybody else.

[Awesome_for_beginners.txt](https://github.com/nebrelbug/awesome-for-beginners/files/1926593/Awesome_for_beginners.txt)

**Comment by bgub** [comment]
Sounds great!

**Comment by bgub** [comment]
Create a pull request with what you have if you think it looks good or want feedback.

**Comment by Meikay** [comment]
@nebrelbug I did the html part, I just want to finish designing it. Once I'm finished I'll make a PR.

**Comment by JohnSquare** [comment]
Perhaps we could start with a briefing document...right?

**Comment by taruchit** [comment]
Hello, is anything left to be done which I can take up?

**Comment by bgub** [comment]
@taruchit That would be great! Here are some possible ideas:

- Improve the styling of the website
- Write a script to automatically fetch the Readme from Github and parse it, then display it
- Add a link to the Github Page

Thanks for your interest!

**Comment by bgub** [comment]
For now, we're using GitHub Pages, thanks to the most recent pull request by Nimish Mishra. For any potential contributors, remember to enable GitHub Pages on your repository so we can preview your changes.

**Comment by taruchit** [comment]
> @taruchit That would be great! Here are some possible ideas:
> 
> * Improve the styling of the website
> * Write a script to automatically fetch the Readme from Github and parse it, then display it
> * Add a link to the Github Page
> 
> Thanks for your interest!

Hello,
As you suggested to improve the styling of the website, we see a lot of content and thus got to scroll down to data under all headers. Thus, can we make a horizontal navigation bar with all headers like .NET, C, C++ and so on. Thus, the home page will get free from all content and visitor can read the required content using navigation bar. Below the navigation bar, we can have images with logo of all languages in carousel fashion. 

I will try to upload here screenshots of the above changes in coming days for your review. 
Please help with your suggestions about the above points.
